src/views/Auction/components/Timer/index.spec.tsx

```diff
@@ -40,67 +40,35 @@ describe('converts unix seconds into local Date time format function', () => {
     })
 })
 
-// variables
-const addHours = (dayjsInstance: Dayjs, hours: number) => dayjsInstance.clone().add(hours, 'h')
-const utcDate = dayjs(new Date().toUTCString())
-const dateUTC = dayjs.unix(utcDate.unix())
+
 
 describe('Timer', () => {
   test('when auction is open it should render the correct display', async () => {
-    const auction = {
-      id: '0x141',
-      startBlock: addHours(dateUTC, -24).unix(),
-      endBlock: addHours(dateUTC, +24).unix(),
-      tokenAddress: '0x',
-      tokenAmount: 5000,
-      tokenName: 'Omen',
-      tokenSymbol: 'DXD',
-      tokenIcon: Omen,
-      bids: [
-        {
-          address: '0xf1',
-          buyAmount: BigNumber.from(30),
-          sellAmount: BigNumber.from(100),
-        },
-      ],
-    }
+    const auction = getAuctionDefault({
+      startBlock: addHours(dateUTC, -0.01).unix(),
+      endBlock: addHours(dateUTC, 0.05).unix()
+    })
+
     const { getByTestId } = render(<Timer auction={auction} />)
 
-    expect(await getByTestId('open')).toHaveTextContent('1d')
+    expect(await getByTestId('open')).toHaveTextContent('3m')
   }),
     test('when auction is upcoming, it displays the correct return', () => {
-      const auction = {
-        id: '0x143',
+      const auction = getAuctionDefault({
         startBlock: addHours(dateUTC, 14).unix(),
-        endBlock: addHours(dateUTC, 114).unix(),
-        tokenAddress: '0x',
-        tokenAmount: 150000,
-        tokenName: 'Compound',
-        tokenSymbol: 'COMP',
-        tokenIcon: Compound,
-        bids: [],
-      }
+        endBlock: addHours(dateUTC, 114).unix()
+      })
+
       const { getByText } = render(<Timer auction={auction} />)
       expect(getByText('to')).toBeInTheDocument()
     }),
     test('when auction is closed, it should return the correct display', async () => {
-      const auction = {
-        id: '0x1434',
+      const auction = getAuctionDefault({
         startBlock: addHours(dateUTC, -140).unix(),
-        endBlock: addHours(dateUTC, -14).unix(),
-        tokenAddress: '0x',
-        tokenAmount: 1500,
-        tokenName: 'Closed',
-        tokenSymbol: 'CLOSE',
-        tokenIcon: Compound,
-        bids: [
-          {
-            address: '0xf1',
-            buyAmount: BigNumber.from(30),
-            sellAmount: BigNumber.from(100),
-          },
-        ],
-      }
+        endBlock: addHours(dateUTC, -14).unix()
+      })
+
+
       const { getByTestId } = render(<Timer auction={auction} />)
       expect(await getByTestId('closed')).toHaveTextContent('GMT')
```

Comment:
Please add exact matching tests.

Reply:
I'm not sure what you mean here. Do you mean that there is a lack of coverage?

Reply:
You only checked the text 'GMT'. Can't you get the exact text?